Library/Formula/hfstospell.rb

```diff
@@ -0,0 +1,23 @@
+class Hfstospell < Formula
+  desc "Helsinki Finite-State Technology ospell"
+  homepage "http://www.ling.helsinki.fi/kieliteknologia/tutkimus/hfst/"
+  url "https://downloads.sourceforge.net/project/hfst/hfst/archive/hfstospell-0.3.0.tar.gz"
+  sha256 "07b5b368882cac2399edb1bb6e2dd91450b56f732c25413a19fcfe194342d70c"
+
+  depends_on "pkg-config" => :build
+  depends_on "libarchive"
+
+  def install
+    # Libxml++ headers do not seem to compile on El Capitan
```

Comment:
Good point. When you know what to look for, documentation says libxml++ needs C++11 but hfts makefiles don't add C++11 flags by default.